src/extensions/lib/src/check-firmware-versions.ts

```diff
@@ -8,16 +8,17 @@ type ProbeInfo = {
 	nodeVersion: string | null;
 }
 
-export const getFirmwareSubject = (version: string) => `Outdated probe firmware, should be ${version}`;
-export const getNodeVersionSubject = (version: string) => `Outdated probe node.js version, should be ${version}`;
+export const OUTDATED_FIRMWARE_NOTIFICATION_TYPE = 'outdated_firmware';
```

Comment:
Since the wording is subject and text are the same, the type should be as well, otherwise it could appear duplicated to the user. That's why I suggested a json type for the other column, to make it more flexible.

Reply:
Ok, but using json type blocks us from using directus filtering rules: https://github.com/directus/directus/discussions/8331, concatenated string with two versions should work fine here.

Reply:
That might be fine too. Though I'm not sure if we need filter rules for this field.

Reply:
There are two cases when we check to send the notification:
1. Immediately after adoption.
2. Cron every day.

In second case, if only filter by type, some notification from year ago may block sending a new one, so I want to filter by versions string additionally, thus sending notification on every target version change (if probe is still outdated ofc).

Reply:
Of course, I thought it would only affect front-end filtering. On the backend I thought we have more flexibility in how the query is made. Either way a string is fine too.